### ISSUE TITLE
Suggestion: Menu Colour

### ISSUE THREAD

**Issue by hackercat27** [absoIute/Mega-Hack-Pro-Future#271]
Why not add an option to change the accent colour of the megahack menu?
Perhaps there could also be some basic options to animate the colour like you can with icon colours (ie: RGB colour cycling, fading between 2 or more colours, etc)

**Comment by absoIute** [comment]
will add specific combo colours when I add theming, but not 100% full customisation since people will just choose colours that don't contrast well and complain about it, though animating the colours isn't too feasible (without changing a lot) due to how the UI engine works sadly

**Comment by Brittank88** [comment]
I get you, though... I kinda don't see why people would complain about something they literally just consciously set themselves. If you're concerned, do what some theme-changers do where it's a "Confirm in the next X seconds!" button.

**Comment by absoIute** [comment]
That's true could do, you'd just be surprised how many times people have come to me after breaking their game by enabling something they shouldn't have

**Comment by Brittank88** [comment]
#553 mentioned an additional feature of this whereby you can apply an animated RGB akin to the icon RGB. Generally speaking, I figure all the colour settings available for icons should be available for menu colourisation.

**Comment by Brittank88** [comment]
@absoIute Could we add the considering label to this? I really don't think people are gonna be so hopeless that they somehow have to complain about their menu colour scheme before just... fixing it lol.

**Comment by defaultacc30** [comment]
> @absoIute Could we add the considering label to this? I really don't think people are gonna be so hopeless that they somehow have to complain about their menu colour scheme before just... fixing it lol.

i think you should just stop leaving for the minority. let them complain. let them be dumb.

**Comment by absoIute** [comment]
I'd rather my mods be straight-forward and less easy to mess-up accidently. When you've got so many options on display you can't blame people for not being able to know exactly what messed up what

**Comment by Brittank88** [comment]
> > @absoIute Could we add the considering label to this? I really don't think people are gonna be so hopeless that they somehow have to complain about their menu colour scheme before just... fixing it lol.
> 
> i think you should just stop leaving for the minority. let them complain. let them be dumb.

The hypocrisy of suggesting we ignore a complaining minority is hilarious lmao
